.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,76 @@
+name: aws-deploy
+
+on:
+  pull_request:
+    branches: '*'
+  push:
+    branches: [main, develop]
+
+jobs:
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  sceptre-deploy:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    needs: pre-commit
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install pipenv
+        run: |
+          pip install pipenv
+      - name: Cache Pipenv virtualenv
+        uses: actions/cache@v2
+        id: pipenv-cache
+        with:
+          path: ~/.local/share/virtualenvs
+          key: ${{ runner.os }}-pipenv-v2-${{ hashFiles('**/Pipfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pipenv-v2-
+      - name: Install Dependencies
+        if: steps.pipenv-cache.outputs.cache-hit != 'true'
+        env:
+          PIPENV_NOSPIN: 'true'
+          WORKON_HOME: ~/.local/share/virtualenvs
+          PIPENV_CACHE_DIR: ~/.local/share/pipcache
+        run: pipenv install --dev
+      - name: Assume AWS Role (Dev Account)
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.CI_SERVICE_USER_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::035458030717:role/SceptreBuildAccessRole
+          role-duration-seconds: 1200
+      - name: Deploy 'dev' and 'prod' Stack Groups to Dev Account
```

Comment:
Why are you launching both dev and prod to the Dev account? Was expecting dev -> dev and prod -> prod.

Reply:
I explained my thinking for this approach [here](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/pull/1#issue-612835313), but I rather not cause any confusion. So, I'll change this to use the approach you described. If the need arises in the future, we can add a `common` stack group for resources we want deployed in both accounts.